docs-2.0/7.data-security/2.backup-restore/1.what-is-br.md

```diff
@@ -0,0 +1,69 @@
+# What is Backup & Restore
+
+Backup & Restore (BR in short) is a Command-Line Interface (CLI) tool for you to back up data of graph spaces of Nebula Graph and to restore data from the backup files.
+
+## Implementation
+
+### Backup
+
+To back up data, the BR sends a backup request to each metad process to trigger the backup process as follows:
+
+1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if remote store such as Amazon S3 or Alibaba Cloud OSS is necessary, their client installation and configuration is verified.
```

Comment:
How to understand this sentence?
The SSH **(has login from the BR to the meta)** and the storage servers is verified
The SSH login from the BR to **(the meta and the storage servers)** is verified
is -> are?